src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx

```diff
@@ -27,6 +32,8 @@ error.type:ConnectionError -> system-down
 
 error.value:"connection error: *" -> connection-error, {{ transaction }}
 ```
+Now, all the events with the error type `DatabaseUnavailable` or `ConnectionError` will be grouped into an issue with the type `system-down`. All events with the error value `connection error` will be grouped into an issue with a type of the same name and the name of the transaction appended.
+<!-- is what i'm saying about transaction correct here? not sure -->
```

Comment:
@untitaker @jjbayer - is what i've said here about the error.value:connection error (last) example true?

Reply:
first sentence is correct, but the behavior for the last rule is that all matching events will be grouped by transaction name.

So if your transaction /api/users/foo/ crashes with a value "connection error", and your transaction /api/events/foo/ crashes the same way, that'll be exactly two issues regardless of stacktrace or whatever else we would normally group by